src/components/plans/ChargeAccordion.tsx

```diff
@@ -217,6 +217,21 @@ export const ChargeAccordion = memo(
       )
     }, [initialLocalCharge?.minAmountCents])
 
+    useEffect(() => {
+      const payInAdvance = localCharge.payInAdvance
+
+      if (payInAdvance === true) {
+        formikProps.setFieldValue(`charges.${index}.minAmountCents`, undefined)
+
+        if (localCharge.chargeModel === ChargeModelEnum.Graduated) {
+          formikProps.setFieldValue(`charges.${index}.prorated`, false)
+        }
+      } else {
+        // Pay in arrears
+        formikProps.setFieldValue(`charges.${index}.invoiceable`, true)
+      }
+    }, [formikProps, index, localCharge.chargeModel, localCharge.payInAdvance])
```

Comment:
@ansmonjol Because `handleUpdate` is passed to component through `onChange` props directly to form components but RadioGroup**Field** uses Formik and does not allow this props so i couldn't call it like this

Reply:
Make sense ok! Thanks for clarification